dip-0011.md

```diff
@@ -134,7 +134,7 @@ Base58 is URL compatible and it is also less prone to spoofing.
 
 # Identity Life Cycle
 
-The life cycle of an Identity is built with multiple different underlying state
+The life cycle of an Identity is composed of multiple different underlying state
```

Comment:
I think the life cycle is a composition of states that are built with state transitions.

Reply:
@shumkov

Reply:
I agree with @QuantumExplorer